### ISSUE TITLE
Validate that schedule content times are reasonable

### ISSUE THREAD

**Issue by jellybob** [emfcamp/Website#1657]
I just spent a bit of time chasing down why stage training was showing up on the wrong day because I'd entered 2025 as the year rather than 2024, it would have been nice to have a validation error telling me I'm trying to schedule content outside of when the event is running.

**Comment by Jonty** [comment]
This will be resolved by the content time periods work that didn't get finished in time, but we implemented the sense checker this year that flags things like this up: https://www.emfcamp.org/cfp-review/sense_check

**Comment by marksteward** [comment]
Depends on #1150
